templates/user/dashboard/milestones.tmpl

```diff
@@ -51,7 +51,7 @@
 							<input type="hidden" name="repos" value="[{{range $.RepoIDs}}{{.}},{{end}}]">
 							<input type="hidden" name="sort" value="{{$.SortType}}">
 							<input type="hidden" name="state" value="{{$.State}}">
-							<input name="q" value="{{$.Keyword}}" placeholder="{{.locale.Tr "explore.search"}}...">
+							{{template "shared/searchinput" dict "." . "Value"}}
```

Comment:
Does the `shared/searchinput` really helps the readability?

I think the old code (HTML tags) are pretty clear.

Reply:
The benefit of the shared template is you do changes in 1 place instead of 17. The `maxlength` change for example was done on only 1 input while all would have benefitted.